assets/js/hooks/useChecks.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * `useChecks` hook tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { renderHook, actHook as act } from '../../../tests/js/test-utils';
+import { muteFetch } from '../../../tests/js/utils';
+import { useChecks } from './useChecks';
+
+describe( 'useChecks', () => {
+	it( 'should return { complete:true, error: undefined } successful check runs.', async () => {
+		muteFetch( /^\/google-site-kit\/v1\/core\/site\/data\/connection/ );
```

Comment:
Again, this hook should be completely decoupled from connection/other state and rely on the given `check` functions entirely.

Reply:
Updated.